modmesh/core.py

```diff
@@ -96,7 +98,7 @@
     'Bezier3dFp64',
     'WorldFp32',
     'WorldFp64',
-    'testhelper',
+    'testhelper'
```

Comment:
There should be a comma after each line in a Python list.  Please be careful about it next time.

Reply:
Thanks, I will be careful about this next time.